From error27@gmail.com  Mon Mar 15 16:32:29 2010
From: Dan Carpenter <error27@gmail.com>
Date: Sat, 6 Mar 2010 14:08:56 +0300
Subject: USB: goku_udc: remove potential null dereference
To: David Brownell <dbrownell@users.sourceforge.net>
Cc: Greg Kroah-Hartman <gregkh@suse.de>, Matthias Kaehlcke <matthias@kaehlcke.net>, Harvey Harrison <harvey.harrison@gmail.com>, Németh Márton <nm127@freemail.hu>
Message-ID: <20100306110856.GF4958@bicker>
Content-Disposition: inline


"dev" is always null here.  In the end it's only used to get the
pci_name() of "pdev" which is redundant information and so I
removed it.

Signed-off-by: Dan Carpenter <error27@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/goku_udc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/gadget/goku_udc.c
+++ b/drivers/usb/gadget/goku_udc.c
@@ -1768,7 +1768,7 @@ static int goku_probe(struct pci_dev *pd
 	 * usb_gadget_driver_{register,unregister}() must change.
 	 */
 	if (the_controller) {
-		WARNING(dev, "ignoring %s\n", pci_name(pdev));
+		pr_warning("ignoring %s\n", pci_name(pdev));
 		return -EBUSY;
 	}
 	if (!pdev->irq) {
